Yank/tests/test_repex.py

```diff
@@ -111,7 +111,7 @@ def test_replica_exchange(mpicomm=None, verbose=True):
 
     """
 
-    if verbose and ((not mpicomm) or (mpicomm.rank==0)): print "Testing replica exchange facility with harmonic oscillators: ",
+    if verbose and ((not mpicomm) or (mpicomm.rank==0)): print("Testing replica exchange facility with harmonic oscillators: ", end = " ")
```

Comment:
Alternatively, we can just use this idiom for the meantime:
```python
sys.stdout.write(string)
```
which doesn't include a newline by default.